src/search_engines.js

```diff
@@ -1344,6 +1344,50 @@ class Github {
   }
 }
 
+class Gitlab {
+  constructor(options) {
+    this.options = options;
+  }
+  get urlPattern() {
+    return /^https:\/\/gitlab-ext\.exxcellent\.de/;
+  }
+
+  get searchBoxSelector() {
+    return '.form-input, input[id=search]';
+  }
+
+  getTopMargin(element) {
+    return getFixedSearchBoxTopMargin(
+        document.querySelector('header.navbar'),
+        element,
+    );
+  }
+
+  onChangedResults(callback) {
+    const containers = document.querySelectorAll('#project-details');
+    const observer = new MutationObserver(async (mutationsList, observer) => {
+      callback(true);
+    });
+    for (const container of containers) {
+      observer.observe(container, {
+        attributes: false,
+        childList: true,
+        subtree: true,
+      });
+    }
+  }
+
+  getSearchResults() {
+    const includedElements = [
+      {
+        nodes: document.querySelectorAll('li.project-row h2 a'),
+        highlightClass: 'wsn-google-focused-link::before',
```

Comment:
Please add a separate gitlab-specific highlight class (even if it's identical)

Reply:
done